sql/core/src/test/scala/org/apache/spark/sql/scripting/SqlScriptingExecutionNodeSuite.scala

```diff
@@ -35,6 +35,10 @@ class SqlScriptingExecutionNodeSuite extends SparkFunSuite with SharedSparkSessi
     override def reset(): Unit = ()
   }
 
+  case class TestLeaveStatement(labelText: String) extends LeaveStatementExec(labelText)
+
+  case class TestIterateStatement(labelText: String) extends IterateStatementExec(labelText)
```

Comment:
Why can't we use the real `Leave/IterateStatementExec` in this test?

Reply:
because I'm dumb :) Used these in `extractStatementValue` to match on `labelText` directly instead of matching by type only and using `Leave/IterateStatementExec`... will change in the next commit to use regular exec nodes

Reply:
Done.